src/AccessibilityInsights.Extensions.GitHub/PlaceHolderTextBox.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    public class PlaceHolderTextBox : TextBox
+    {
+        private readonly string PlaceHolder = "https//github.com/owner/repo";
+        private readonly SolidColorBrush BlackBrush = Application.Current.Resources["TextBrush"] as SolidColorBrush;
+        private readonly SolidColorBrush GrayBrush = Application.Current.Resources["TextBrushGray"] as SolidColorBrush;
+
+        public Action UpdateSaveButton { get; set; }
+
+        public PlaceHolderTextBox()
+        {
+            this.Text = this.PlaceHolder;
+            this.Foreground = GrayBrush;
+
+            this.GotFocus += RemoveText;
```

Comment:
it would be good to double-check this experience on a screen reader to see if the placeholder is read

Reply:
I don't think, it will read the Automation name, placeholders usually get removed once they get focus so the screen readers never read it